gap/tools.gi

```diff
@@ -109,10 +109,12 @@ InstallMethod( IsCompatiblePolyhedronList,
     
     fi;
     
-    if not ( Length( list[ 7 ]  ) = 2 * list[ 4 ] * list[ 5 ] ) then
-      
-      return Error( "The matrix is not compatible" );
+    if NrRows( list[ 7 ] ) <> list[ 4 ] then
```

Comment:
By using NrRows, this code actually requires GAP >= 4.9 (see also <https://github.com/kamalsaleh/CddInterface/issues/11>). Of course I can avoid it, at the cost of readability. But I think requiring GAP 4.9 is reasonable -- agreed?

Reply:
Now the package requires Gap >= 4.9. So, it adjustment is fine.